CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# Contributing Guide
+
+* [New Contributor Guide](#contributing-guide)
```

Comment:
```suggestion
* [Contributing Guide](#contributing-guide)
```
Were those generated automatically? Only this line is different from actual title.